src/WebJobs.Script/Binding/ExtensionLoader.cs

```diff
@@ -1,128 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Reflection;
-using Microsoft.Azure.WebJobs.Host;
-using Microsoft.Azure.WebJobs.Host.Config;
-using Microsoft.Azure.WebJobs.Script.Description;
-using Microsoft.Azure.WebJobs.Script.Models;
-using Microsoft.Extensions.Logging;
-using Newtonsoft.Json.Linq;
-using FunctionMetadata = Microsoft.Azure.WebJobs.Script.Description.FunctionMetadata;
-
-namespace Microsoft.Azure.WebJobs.Script.Binding
-{
-    internal class ExtensionLoader
```

Comment:
We no longer grovel through assemblies looking for IExtensionConfigProviders and [Binding] annotated parameters. Extensions are loaded solely based on the startup extensions declared in extensions.json